spec/features/admin_can_see_trash_icon_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+feature "AdminCanSeeTrashIcon", type: :feature do
```

Comment:
You are testing this behavior in AdminCanDeleteUser specs, so it is not necessary and can be deleted.

If you can delete a user it means that you can see trash icon.